src/genie/libs/parser/slxos/show_version.py

```diff
@@ -0,0 +1,203 @@
+            'install_time': str,
+            'system_uptime': str,
+            Optional('primary_ver'): str,
+            Optional('secondary_ver'): str,
+            Optional('slot'): {
+                Any(): {
+                    'name': str,
+                    'status': str,
+                    'primary_ver': str,
+                    'secondary_ver': str
+                }
+            }
+        }
+    }
+
+
+# ===================================
+# Parser for:
+#   * 'show version'
+# ===================================
+class ShowVersion(ShowVersionSchema):
+    ''' Parser for "show version" '''
+
+    cli_command = ['show version']
+    """
+    Copyright (c) 1995-2021 Extreme Networks, Inc.
+    Firmware name:      20.2.2a
+    Build Time:         10:29:46 Nov 11, 2020
+    Install Time:       05:00:38 Nov 28, 2020
+    Kernel:             4.14.67
+    Control Processor:  Intel(R) Xeon(R) CPU D-1527 @ 2.20GHz,  4 cores
+    Microcode Version:  0x7000017
+    Memory Size:        System Total: 31644 MB
+    System Uptime:      15days 14hrs 36mins 57secs
+
+    Name     Primary/Secondary Versions
+    ------------------------------------------
+    SLX-OS   20.2.2a
+             20.2.2a
+    """
+    """
+    SLX-OS Operating System Software
+    SLX-OS Operating System Version: 18r.1.00
+    Copyright (c) 1995-2021 Extreme Networks Systems, Inc.
+    Firmware name:      18r.1.00h
+    Build Time:         08:13:43 Apr  7, 2021
+    Install Time:       07:00:06 May 18, 2021
+    Kernel:             2.6.34.6
+
+    Control Processor:  GenuineIntel
+    Memory Size:        SLXVM: 11468 MB   System Total: 32174 MB
+
+    System Uptime:   93days 5hrs 32mins 10secs
+
+    Slot    Name    Primary/Secondary Versions                         Status
+    ---------------------------------------------------------------------------
+    M1      SLX-OS  18r.1.00h                                          STANDBY
+                    18r.1.00h
+    M2      SLX-OS  18r.1.00h                                          ACTIVE*
+                    18r.1.00h
+    L1/0    SLX-OS  18r.1.00h                                          ACTIVE
+                    18r.1.00h
+    L2/0    SLX-OS  18r.1.00h                                          ACTIVE
+                    18r.1.00h
+    L3/0    SLX-OS  18r.1.00h                                          ACTIVE
+                    18r.1.00h
+    L4/0    SLX-OS  18r.1.00h                                          ACTIVE
+                    18r.1.00h
+    L5/0    SLX-OS  18r.1.00h                                          ACTIVE
+                    18r.1.00h
+    L6/0    SLX-OS  18r.1.00h                                          ACTIVE
+                    18r.1.00h
+    L7/0    SLX-OS  18r.1.00h                                          ACTIVE
+                    18r.1.00h
+    L8/0    SLX-OS  18r.1.00h                                          ACTIVE
+                    18r.1.00h
+    """
```

Comment:
Please remove these lines of device output as we already have device output in the tests and in the comments